packages/ui/src/components/sheet/sheet.native.tsx

```diff
@@ -0,0 +1,34 @@
+import { NativeScrollEvent, NativeSyntheticEvent } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+
+interface SheetProps {
+  children: React.ReactNode;
+  contentContainerStyle?: Record<string, unknown>;
+  contentInset?: Record<string, unknown>;
+  horizontal?: boolean;
+  onScroll?: (event: NativeSyntheticEvent<NativeScrollEvent>) => void;
+  style?: Record<string, unknown>;
+}
+
```

Comment:
`Sheet` is basically just a wrapper of `ScrollView`. I'm not sure if we even need to do that. 

Sometimes we pass more props to `ScrollView`, for example I had to add `contentInset` + `onScroll`. 

It helps us to link it to `Sheet` in design but I'm not sure if it's better than just consuming `ScrollView`. 

It might make more sense having it here when the footers are implemented https://github.com/leather-io/issues/issues/231

Reply:
@edgarkhanzadian can you provide feedback here on what you've experienced working with RN, is it helpful to wrap `ScrollView` like this so it is coordinated with designs and the extension? It does make sense to me to do, but I'm also not sure if it is overkill?